packages/kit/src/views/Developer/pages/Gallery/Components/stories/TableGallery/components/types.ts

```
@@ -0,0 +1,22 @@
+  id: string;
+  symbol: string;
+  name: string;
+  price: number;
+  change24h: number;
+}
+
+export interface IFeatureData {
+  id: string;
+  feature: string;
+  supported: string;
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)